internal/data/receivers.go

```diff
@@ -31,6 +31,9 @@ type ReceiverRegistrationRequest struct {
 	VerificationValue string            `json:"verification"`
 	VerificationType  VerificationField `json:"verification_type"`
 	ReCAPTCHAToken    string            `json:"recaptcha_token"`
+	ExternalID        string            `json:"external_id"`
+	CustomerID        string            `json:"customer_id"`   //customer identifier
```

Comment:
One of these will eventually be removed once we nail the customer_id vs external_id discussion.

Reply:
Oh, I can see that [SEP-24 uses `customer_id`](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0024.md#request), so we should use the same wording.

```suggestion
	// CustomerID is the SEP-24 field that we store in our database's `receivers.external_id`.
	CustomerID        string            `json:"customer_id"`   //customer identifier
```